Vagrantfile

```diff
@@ -58,6 +58,7 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.synced_folder project_src_dirs, "/home/vagrant/src", create: true
+  config.vm.synced_folder File.join(project_src_dirs,"go_src"), "/home/vagrant/go", create: true
```

Comment:
Just a nitpick: why not `File.join(project_src_dirs,"go")` ?

Since it is possible that any `go get` commands within the devenv would also add `bin` and `pkg` directories. Is having `go_src/bin` weird for a dir that isn't purely source code, or is that just me?

Reply:
`File.join(project_src_dirs,"go")` sounds good